services/ui-auth/serverless.yml

```diff
@@ -77,6 +84,52 @@ resources:
         ExplicitAuthFlows:
           - ADMIN_NO_SRP_AUTH
         GenerateSecret: false
+        AllowedOAuthFlows:
+          - implicit
+        AllowedOAuthFlowsUserPoolClient: true
+        AllowedOAuthScopes:
+          - email
+          - openid
+          - aws.cognito.signin.user.admin # Needed to call Auth.currentUserInfo()
+        CallbackURLs:
+          - ${self:custom.application_endpoint_url}
+          - http://localhost:3000/
```

Comment:
suggest commenting these `localhost:3000` references out, so that they're still visible to the user as a suggestion but won't accidentally make it to higher envs without action.

Reply:
Got it.  I'd like to leave this be for the moment, as hooking into cognito from your local react requires it, and I want to avoid breaking that workflow.  This should be removed from higher environments, though, and some logic could be added to make it so if we think its worthwhile.